ktor-server/ktor-server-core/jvm/src/io/ktor/routing/RouteSelector.kt

```diff
@@ -225,43 +223,40 @@ public data class OptionalParameterRouteSelector(val name: String) :
  * Evaluates a route against a constant path segment
  * @param value is a value of the path segment
  */
-public data class PathSegmentConstantRouteSelector(val value: String) : RouteSelector(RouteSelectorEvaluation.qualityConstant) {
-
-    private var hasTrailingSlash: Boolean = false
+public data class PathSegmentConstantRouteSelector(val value: String) :
```

Comment:
codestyle

Reply:
What is the correct codestyle here? It doesn't fit in one line. Will you prefer
```
public data class PathSegmentConstantRouteSelector(val value: String) : RouteSelector(
    RouteSelectorEvaluation.qualityConstant
) {
```

Reply:
Yep, it's correct according: https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting

There is different option:
```kotlin
public data class PathSegmentConstantRouteSelector(
val value: String
) : RouteSelector(RouteSelectorEvaluation.qualityConstant) {
```